server/handler.go

```diff
@@ -401,7 +401,7 @@ func (h *Handler) SetStoreLimit(storeID uint64, rate float64) error {
 	if err != nil {
 		return err
 	}
-	c.opController.SetStoreLimit(storeID, rate)
+	c.opController.SetStoreLimit(storeID, rate, schedule.StoreLimitManual)
```

Comment:
why to set manual mode when get all store limit?

Reply:
This is the HTTP API, when it is called, we assume that the conf is set by the user.

Reply:
I think it is strange to change mode when only to read some config.

Reply:
What do you mean about reading some config? Is it a mistake that you regard `SetStoreLimit` as `GetAllStoresLimit`? Expanding the code between lines `393` and `401` would be helpful.